packages/did/src/Did.chain.ts

```diff
@@ -515,21 +516,23 @@ export async function fetchFromMnemonic(mnemonic: string): Promise<DidDocument>
  */
```

Comment:
Updating the method docs would be good to have.
Mention `keytype`, `mnemonic` is optional.